index.html

```diff
@@ -100,21 +100,21 @@ <h2 id="examples">
         }
       </pre>
       <p>
-        This example is checking whether the page has the {{PermissionName/"geolocation"}} and the
-        {{PermissionName/"notifications"}} permissions:
+        This example simultaneously checks the state of the `"geolocation"` and `"notifications"`
+        [=powerful features=]:
       </p>
       <pre class="example js" title="Checking the state of multiple permissions">
-        const queryPromises = ["geolocation", "notifications"].map(name =&gt; {
-          return navigator.permissions.query({ name });
-        });
+        const queryPromises = ["geolocation", "notifications"].map(
+          name =&gt; navigator.permissions.query({ name });
```

Comment:
```suggestion
          name =&gt; navigator.permissions.query({ name })
```

Reply:
(that semi-colon will cause a syntax error because we're inside a fat-arrow fn w/o braces)